docs/TESTS.md

```diff
@@ -81,6 +81,20 @@ Good luck! Have fun!
 
    _(Don't worry about the tests failing, at first, this is how you begin each exercise.)_
 
+   If you get the following error:
+
+   ```sh
+   ./gradlew: Permission denied
+   ```
+
+   Then that means there's an issue regarding permissions, that the file isn't executable. To fix this, run:
```

Comment:
I wanted to suggest a more direct wording for the first sentence. What do you think? 

```suggestion
   Then the file is missing the execute permission. To fix this, run:
```

Reply:
I agree, this wording seems better.